node/node.go

```diff
@@ -649,7 +685,10 @@ func (node *Node) handleNewMiningOutput(miningOutCh <-chan mining.Output) {
 
 }
 
-func (node *Node) handleNewHeaviestTipSet(ctx context.Context, head types.TipSet) {
+func (node *Node) handleNewChainHeads(ctx context.Context, prevHead types.TipSet) {
+	node.HeaviestTipSetHandled = func() {}
```

Comment:
This appears to be unused here and in tests. If so it would be nice to just remove it.

Reply:
Oh wonderful, thanks. Re https://github.com/filecoin-project/go-filecoin/issues/2309